src/style.css

```diff
@@ -0,0 +1,134 @@
+/* #region Normalize */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --blue-color: #00acdc;
+  --text-dark-color: #060b35;
+  --text-light-color: #616070;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+/* #endregion */
+
+/* #region reset */
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--text-dark-color);
+}
+
+.card__product {
+  margin-bottom: 16px;
+  color: var(--text-light-color);
+}
+
+.card__rating-review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  text-align: left;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+.card__review {
+  color: var(--text-dark-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
flexbox is not for this purpose, it's useful when we have multiple items to arrange